relation_engine_server/test/data/json_validation/invalid_array_items.yaml

```diff
@@ -0,0 +1,6 @@
+name: invalid_array
```

Comment:
Why test both yaml and json for everything? I could see that making sense for the date type or some other yaml quirk, but for anything else it seems overboard.

Reply:
The test helper I'm using automatically tests the different file types, and it takes very little time to generate JSON from YAML. May as well be consistent with all the tests.